File: ToDoApp_backend/ViewModel/ScheduleList/ScheduleListViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ToDoApp_backend.ViewModel.ScheduleList
{
    public class ScheduleListViewModel
    {
        public long ScheduleId { get; set; }
        public string ScheduleName { get; set; }
        public string Summary { get; set; }
        public string StartDay { get; set; }
        public string EndDay { get; set; }
        public string UsageTime { get; set; }
        //値保存用
        public long CategoryMasterId { get; set; }
        public string CategoryMasterName { get; set; }

        //削除フラグ(フロント用)
        public bool DeletedFlg { get; set; }
    }
}

File: frontend_App/todo-list/types/ViewModel/ScheduleListViewModel.ts
export interface IScheduleListViewModel {
  ScheduleId: number
  ScheduleName: string // スケジュール名
  Summary: string // 概要説明
  StartDay: string // 開始日
  EndDay: string // 終了日
  UsageTime: string // 使用時間
  CategoryMasterId: number
  CategoryMasterName: string
  DeletedFlg: boolean
}

export class ScheduleListViewModel implements IScheduleListViewModel {
  ScheduleId = 0
  ScheduleName = ''
  Summary = ''
  StartDay = ''
  EndDay = ''
  UsageTime = ''
  CategoryMasterId = 0
  CategoryMasterName = ''
  DeletedFlg = false
}
